server/api.go

```diff
@@ -724,7 +725,17 @@ func (a *API) handleSyncNotificationsWelcomeMessage(originInfo string, mmUserID,
 				a.p.API.LogWarn("Unable to update post", "post", postID, "error", appErr.Error())
 			}
 		} else {
-			a.p.GetAPI().DeleteEphemeralPost(mmUserID, postID)
+			// Update the original connection prompt and remove any calls to action.
```

Comment:
Make sure the user comes back to Mattermost with an ephemeral message explaining what happened. We also send a welcome message from the bot itself.